Tools/Create-Azure-Sentinel-Solution/input/Solution_DigitalShadowsSearchlight.json

```
@@ -0,0 +1,15 @@
+ {
+    "Name": "Digital Shadows Searchlight",
+    "Author": "Digital Shadows - support@digitalshadows.com",
+    "Logo": "<img src=\"https://raw.githubusercontent.com/parag-metron/Azure-Sentinel/Digital-Shadows-connector/Logos/DigitalShadowsLogo.svg\" width=\"75px\" height=\"75px\">",
+    "Description": "{Solution for Azure Sentinel using Digital Shadows Searchlight.}",
+    "WorkbookDescription": "For gaining insights into Digital Shadows logs.",
+    "Workbooks": ["Workbooks/DigitalShadows.json"],
+    "Analytic Rules": ["Analytic Rules/Digital_Shadows_incident_creation_exclude.yaml", "Analytic Rules/Digital_Shadows_incident_creation_include.yaml"],
+    "Playbooks": ["Playbooks/Solution_DigitalShadowsSearchlight.json"],
+    "Data Connectors": ["Data Connectors/Digital shadows/DigitalShadowsSearchlight_API_functionApp.json"],
```

@shikhin-metron the path doesn't look right here - should be 

```suggestion
    "Data Connectors": ["Data Connectors/Digital Shadows/DigitalShadowsSearchlight_API_functionApp.json"],
```